### ISSUE TITLE
Webpack 5 & false circular dependency warning

### ISSUE THREAD

**Issue by qortex** [apollographql/graphql-tag#326]
I just switched to Webpack 5 (Angular 11) and I now get 1 warning per `.graphql` file:

```
Warning: Circular dependency detected:
src/app/backbone/identity/person.graphql -> src/app/backbone/identity/person.graphql
```

Nothing fancy with my setup I think, just this webpack extra config:
```js
module.exports = {
  module: {
    rules: [
      {
        test: /\.(graphql|gql)$/,
        exclude: /node_modules/,
        loader: 'graphql-tag/loader',
      },
    ],
  },
};
```

My `graphql` files don't import themselves or other files. Just simple graphql queries & mutations.

The warnings are there, but the app works well nonetheless.

**Comment by qortex** [comment]
Closed through https://github.com/just-jeb/angular-builders/issues/841

TLDR; add `"circular-dependency-plugin": "^5.2.2"` to `resolutions` in `packages.json`, if a smaller version is used at the time you read this. The bug is fixed in this version.
